src/data/store/Layers.js

```diff
@@ -227,13 +263,13 @@ Ext.define('GeoExt.data.store.Layers', {
         var layer = evt.element;
         var index = this.layers.getArray().indexOf(layer);
         var me = this;
-        layer.on('propertychange', me.onChangeLayer, me);
         if (!me._adding) {
             me._adding = true;
             var result = me.proxy.reader.read(layer);
             me.insert(index, result.records);
             delete me._adding;
         }
+        me.bindLayer(layer);
```

Comment:
Just for clarification: Why is it necessary to do the binding here again? Shouldn't it be done already?

Reply:
I don't now where? But I have to admit that I do not exactly know how the store 
 
This method is called when a new layer is added to the layers collection. In that case I would assume it was not bind yet?

Should it possible to add a record without adding a layer? I don't think that case is covered at all.

Reply:
Ok, then this is fine. I think I missed that this is done when a new layer is added to the layers collection. Thanks for clarification.